content/en/platform/corda/5.0/application-networks/creating/mgm/cpi.md

```diff
@@ -165,7 +165,7 @@ Build a CPI using the Corda CLI, passing in your generated `GroupPolicy.json` fi
    {{< /tabs >}}
 
 {{< note >}}
-Unlike other CPIs, an MGM CPI does not require a CPB file.
+Unlike other CPIs, an MGM CPI does not require a {{< tooltip >}}CPB{{< /tooltip >}} file.
```

Comment:
don't you need a tooltip for CPI and MGM here ?

Reply:
This section is about uploading the MGM CPI. So these two concepts have already been introduced at the start.
This is something I intend to raise with the team but currently, the plan is that not every single instance of a term would have a tooltip. (Similar to the way we only spell out acronyms in the first instance).

In this particular instance, I have actually removed all tooltips as it seems our site won't render them correctly when in note boxes.

Reply:
I think we need to re-think our methodology.
Is there a different way to employ tooltips w/o manually marking them (e.g. `{{ < tooltip >}}`) ?
I'm thinking about another way in which we have a list of glossary items and when we generate the doc-site (either Hugo or some other way) we do a pre-scan for all glossary items and then mark those in this pre-scan ... then we generate the site ... 
Thoughts ?!

Reply:
My vote would be have a tooltip on every term.

Reply:
I'll raise this with the rest of the team next week but I'm of the opinion that it needs to be manual. I think the interface could become very cluttered if every instance of every term had a tooltip. For example, some terms could appear more than once in the same sentence. Also, we do not want a tooltip in a section that explains a term. For example, "CPK", "CPI", and "CPB" on https://docs.r3.com/en/platform/corda/5.0/developing-applications/packaging.html don't need tooltips.
This is a large initial task now because the glossary terms were defined late in the documentation process. Going forward, new terms should be added as they arise and it should become second nature for writers to know when to call the shortcode when writing content.

Reply:
Too many glossary links may have the same affect as too many hyperlinks and affect readability though: https://idratherbewriting.com/2010/06/19/finally-convinced-about-removing-inline-links-to-increase-readability/

If possible, one ideal would be to automate so only the first occurrence of a term in a topic is set as a glossary link.